actions/class.Items.php

```diff
@@ -219,6 +219,14 @@ public function editItem()
             $this->setData('isAuthoringEnabled', $hasModel);
 
             $this->setData('formTitle', __('Edit Item'));
+            
+            if ($myForm->hasAsyncFileUpload()) {
```

Comment:
We will need this property to work on all sort of extensions that work with metadata. Example. Tests. This fix should be applied on tao-core most probably.

Wdyt @chinnu-focaloid ?